lectures/multi_agent_models/lake_model.md

```diff
@@ -669,31 +686,30 @@ function of the unemployment compensation rate
 ```{code-cell} julia
 function compute_optimal_quantities(c_pretax, tau; w_probs, sigma, gamma, beta,
                                     alpha, w_pretax)
-    mcm = mcall_model(; alpha, beta, gamma, sigma, w_probs,
-                      c = c_pretax - tau, # post-tax compensation
-                      w = w_pretax .- tau)
+    mcm = mccall_model(; alpha, beta, gamma, sigma, w_probs, c = c_pretax - tau,
+                       w = w_pretax .- tau)
 
     (; V, U, w_bar) = solve_mccall_model(mcm)
     accept_wage = w_pretax .- tau .> w_bar
 
     # sum up proportion accepting the wages
-    lambda = gamma * dot(w_probs, accept_wage)
-    return w_bar, lambda, V, U
+    lambda = gamma * dot(mcm.w_probs, accept_wage)
```

Comment:
use `dot(w_probs, accept_wage)` or something like that instead after you splat it as above.